experimenter/experimenter/nimbus-ui/src/components/AppLayoutSidebarLaunched/index.tsx

```diff
@@ -236,6 +237,22 @@ export const AppLayoutSidebarLaunched = ({
                   )}
                 </DisabledItem>
               )}
+
+              <p className="edit-divider position-relative small my-2">
+                <span className="position-relative bg-light pl-1 pr-2 text-muted">
+                  History
+                </span>
+              </p>
+
+              <LinkNav
+                route={`/history/${slug}`}
+                testid={`history-page-${slug}`}
+                isExternalRoute={true}
```

Comment:
No need to add `={true]` for boolean props.
```suggestion
                isExternalRoute
```